### PR TITLE
Optional makeEmpty in THREE.Box3

### PR DESCRIPTION
**Pull request by sunag**
Hi.
This solve problems with to collect parallel objects(no relationship) in THREE.Box3.
Otherwise would have to create a `Box3` for each object or add all in an single `Object3D`

```javascript
var meshes = [mesh1, mesh2, ..[n].. ]
var bbox = new THREE.Box3();

for(var i = 0; i < meshes.length; i++) {
	bbox.setFromObject( meshes[i], false ); // false = not call `makeEmpty`
}
```

### CODE REVIEW

**Comment by WestLangley** [comment]
Instead, why not add the method
```javascript
Box3.expandByObject( object )
```
and rewrite `setFromObject()` to call the new method.

**Comment by sunag** [comment]
> and rewrite setFromObject() to call the new method.

@WestLangley added!

File: src/math/Box3.js [hunk 1]
```diff
@@ -92,8 +94,6 @@ Box3.prototype = {
 
 	setFromPoints: function ( points ) {
 
-		this.makeEmpty();
```
Comment (WestLangley) [inline src/math/Box3.js]: Why did you remove `makeEmpty()` here?

File: src/math/Box3.js [hunk 2]
```diff
@@ -121,69 +121,13 @@ Box3.prototype = {
...
-					} else if ( geometry.isBufferGeometry ) {
-
-						var attribute = geometry.attributes.position;
-
-						if ( attribute !== undefined ) {
-
-							for ( i = 0, l = attribute.count; i < l; i ++ ) {
-
-								v1.fromBufferAttribute( attribute, i ).applyMatrix4( node.matrixWorld );
+	setFromObject: function ( object ) {
```
Comment (Mugen87) [inline src/math/Box3.js]: Shouldn't there be the optional parameter `makeEmpty`?

**Comment by Mugen87** [comment]
I'm not 100% sure about the consistency of this change. `Box3 .setFromObject` will be the only method that allows an optional `empty` operation. But what about the other `.setFrom*` methods?

**Comment by WestLangley** [comment]
@Mugen87 We have `expandBy*()` and `setFrom*()` now. See the coments above, and ignore the post title.

**Comment by Mugen87** [comment]
I see...

File: src/math/Box3.js (hunk 1 above)
Comment (sunag) [inline src/math/Box3.js]: Sry my distraction... I will add.

**Comment by mrdoob** [comment]
Thanks!

**Comment by calvin** [comment]
Need to update the docs for new `.expandByObject()` method.
